packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -142,6 +184,54 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
     this.ariaMessage = message;
   };
 
+  private toggleClass(element: Element, className: string, condition: boolean) {
+    element.classList.toggle(className, condition);
+  }
+
+  private adaptAnswerHeight() {
+    this.fullAnswerHeight = this.host?.shadowRoot
+      ?.querySelector('p[part="generated-text"]')
```

Comment:
I'd suggest to remove the element from the selector. For rich formatting, `generated-text` may either be `<p>` or a `<div>` depending if it is plain text or not.

```suggestion
      ?.querySelector('[part="generated-text"]')
```